### ISSUE TITLE
dem does not work on deno v0.29.0

### ISSUE THREAD

**Issue by uki00a** [syumai/dem#6]
Hello, thank you for the module!

I run into a problem when run `dem` on deno v0.29.0 like this:

```sh
$ dem update https://deno.land/std@v0.29.0
sub command https://deno.land/std@v0.29.0 does not exist.
```

I think this is because `Deno.args`  now does not include script name. (https://github.com/denoland/deno/pull/3628)

Would you have a plan to support deno v0.29.0 or above?

Thank you.

**Comment by syumai** [comment]
Thank you for this PR!
Simply I'll apply this change and release v0.3.0.
By the way, `deno install` seems also broken (because Deno v0.29.0 uses std@v0.28.1) , and I can't install dem now...
I believe it'll be fixed in v0.30.0 😢

**Comment by syumai** [comment]
@uki00a 
I've released this in 0.3.0, Thanks!
https://github.com/syumai/dem/releases/tag/0.3.0

**Comment by uki00a** [comment]
Thanks!
